package.json

```diff
@@ -28,7 +28,7 @@
   "author": "Brigade Engineering",
   "license": "MIT",
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0"
```

Comment:
Is one of the changes in this PR making the code incompatible with 0.14? From what I understand the `prop-types` lib will work fine with 0.14. If that's the case, then maybe we should keep the `peerDeps` open until the need arises to make it more strict?